app/_hub/kong-inc/openid-connect/how-to/_fapi.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Financial-Grade API (FAPI)
+nav_title: Financial-Grade API (FAPI)
+minimum_version: 3.7.x
+---
+
+## Overview
+
+The OpenID Connect plugin supports various features of the FAPI standard, aimed to protect APIs that expose high-value and sensitive data.
+
+### Pushed Authorization Requests (PAR)
+
+With PAR enabled, Kong (OAuth client) sends the payload of an authorization request to the IdP. As a result, it obtains a `request_uri` value. This value is used by the client in a call to the authorization endpoint as a reference to obtain the authorization request payload data.
+
+For more information refer to the [Configuration reference](/hub/kong-inc/openid-connect/configuration/#config-pushed_authorization_request_endpoint)
+
+### JWT-Secured Authorization requests (JAR)
+
+With JAR enabled, when sending requests to the authorization endpoint, Kong provides request parameters in a JSON Web Token (JWT) instead of using a query string, which allows the request data to be signed with JSON Web Signature (JWS).
+
+For more information refer to the [Configuration reference](/hub/kong-inc/openid-connect/configuration/#config-require_signed_request_object)
+
+### JWT-Secured Authorization Response Mode (JARM)
+
+With JARM enabled, Kong requests the authorization server to return the authorization response parameters encoded in a JWT, which allows the response data to be signed with JSON Web Signature (JWS).
+
+JARM can be enabled by setting the `response_mode` configuration option to any of the following available values: `query.jwt`, `form_post.jwt`, `fragment.jwt`, `jwt`.
+
+For more information refer to the [Configuration reference](/hub/kong-inc/openid-connect/configuration/#config-response_mode)
+
+### Certificate-Bound Access Tokens
+
+Certificate-bound access tokens allow binding tokens to clients. This guarantees the authenticity of the token by verifying whether the sender is authorized to use it for accessing protected resources.
+
+For more information refer to the [dedicated section](/hub/kong-inc/openid-connect/how-to/cert-bound-access-tokens).
+
+### Mutual TLS (mTLS) Client Authentication
+
+When mTLS authentication is enabled, Kong establishes mTLS connections with the IdP using the configured client certificate.
```

Comment:
We need to mention that Kong is capable of verifying such a bind of cert (the RP role).

Reply:
yes good point I forgot to include Certificate-Bound Access Tokens